api/v1/sriovnetworknodepolicy_types.go

```diff
@@ -59,6 +59,8 @@ type SriovNetworkNodePolicySpec struct {
 	VdpaType string `json:"vdpaType,omitempty"`
 	// Exclude device's NUMA node when advertising this resource by SRIOV network device plugin. Default to false.
 	ExcludeTopology bool `json:"excludeTopology,omitempty"`
+	// don't create the virtual function only allocated them to the device plugin. Defaults to false.
+	ExternallyCreated bool `json:"externallyCreated,omitempty"`
```

Comment:
how does ExternallyCreated relate to other parameters here ?

we need to have this documented so users will know what to expect when using it.

Reply:
I add documentation for it let me know if you need anything else :)

Reply:
is there a reason to do it per interface and not global? 
Do you have use-case when some SR-IOV will be create by SR-IOV opeartor and some not?

Reply:
I read it now in the description. You want to use it for  CNI primary network and secondary will VF still be managed by sriov operator